langlib/lang.map/src/main/java/org/ballerinalang/langlib/map/ToArray.java

```diff
@@ -43,7 +43,7 @@
  * @since 1.2.0
  */
 @BallerinaFunction(
-        orgName = "ballerina", packageName = "lang.map",
+        orgName = "ballerina", packageName = "lang.map", version = "1.1.0",
```

Comment:
for lang.lib module versions, can we use constants and refer them wherever needed?